ckanext/ontario_theme/templates/internal/package/base.html

```diff
@@ -16,12 +16,7 @@
     {% else %}
       <li>{% link_for _('Datasets'), 'dataset.search' %}</li>
     {% endif %}
-    <li id="dataset-name" dataset-name="{{ pkg.title }}" dataset-name-fr="{{ h.ontario_theme_get_translated_lang(pkg, 'title', 'fr') }}" {{ self.breadcrumb_content_selected() }}>{% link_for dataset|truncate(30), named_route='dataset.read', id=pkg.name %}
```

Comment:
I think this removes the dataset name when editing a resource or adding a new resource to an existing dataset